### DISCUSSION TITLE
How to install AGH offline on Raspberry Pi?

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#4839] [Q&A]
Is it possible to download AGH as a DEB package and use something like "dpkg -i XXX.deb" command to extract it and have AGH service install itself? Or some other way that doesn't require connecting Raspberry Pi to internet until AGH is installed?

**Comment by ainar-g** [comment]
You can simply unpack the archive and install it.  See here: <https://github.com/AdguardTeam/AdGuardHome/wiki/Getting-Started#official-releases>.
